stoe-ichenhausen/ichenhausen.md

```diff
@@ -0,0 +1,5 @@
+:wave: Hi!
+
+![](https://github.com/stoe.png)
+
+I'm **Stefan** and I'm from [Ichenhausen](https://en.wikipedia.org/wiki/Ichenhausen).
```

Comment:
Can you give us an interesting fact from this wiki page so we don't have to open it ourselves? ❤️